### PR TITLE
Create ui-c8y-1019-20-6-fix-software-list-schema-definition-for-properties-library.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [DM-3470](https://cumulocity.atlassian.net/browse/DM-3470)
Original PR: [6080](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6080)

### CODE REVIEW

File: content/change-logs/device-management/ui-c8y-1019-20-6-fix-software-list-schema-definition-for-properties-library.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Fix software list schema definition for properties library
```
Comment (bozhidardedov) [inline content/change-logs/device-management/ui-c8y-1019-20-6-fix-software-list-schema-definition-for-properties-library.md:3]:
```suggestion
title: Fixed "Software list" schema definition for properties library
```

File: content/change-logs/device-management/ui-c8y-1019-20-6-fix-software-list-schema-definition-for-properties-library.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3470
+version: 1019.20.6
+---
+Fix software list schema definition for properties library
```
Comment (bozhidardedov) [inline content/change-logs/device-management/ui-c8y-1019-20-6-fix-software-list-schema-definition-for-properties-library.md:17]:
```suggestion
The properties library schema definition for the `c8y_SoftwareList` fragment did not match the actual managed object properties. Hence, when "Software" property was selected in "Device data" widget no data was shown. The property is now called "Software list" and displays all software items contained in the `c8y_SoftwareList` fragment.
```

File: content/change-logs/device-management/ui-c8y-1019-20-6-fix-software-list-schema-definition-for-properties-library.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3470
+version: 1019.20.6
+---
+The properties library schema definition for the `c8y_SoftwareList` fragment did not match the actual managed object properties. Hence, when "Software" property was selected in "Device data" widget no data was shown. The property is now called "Software list" and displays all software items contained in the `c8y_SoftwareList` fragment.
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-1019-20-6-fix-software-list-schema-definition-for-properties-library.md:17]:
```suggestion
The properties library schema definition for the `c8y_SoftwareList` fragment did not match the actual managed object properties. Hence, when the "Software" property was selected in the "Device data" widget no data was shown. The property is now called "Software list" and displays all software items contained in the `c8y_SoftwareList` fragment.
```

File: content/change-logs/device-management/ui-c8y-1019-20-6-fix-software-list-schema-definition-for-properties-library.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Fixed "Software list" schema definition for properties library
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-1019-20-6-fix-software-list-schema-definition-for-properties-library.md:3]:
```suggestion
title: Software items are now properly shown in the Device data widget
```

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@l2p- I assume this can be merged as well? If so, please approve.

**Review by dafkk — APPROVED** [review]

**Pull request merged**